index/test_bootstrapping.sh

```diff
@@ -4,7 +4,7 @@
 set -ex
 
 docker-compose up -d
-docker-compose run index bash -c "/code/bootstrap-odc.sh \$PRODUCT_CATALOG \$METADATA_CATALOG"
+docker-compose run index bash -c "cd \$HOME && /code/bootstrap-odc.sh \$PRODUCT_CATALOG \$METADATA_CATALOG"
```

Comment:
Might as well change to home in the image, I think... This is safe too, I guess!

Reply:
Yeah, this is not needed with the USER and WORKDIR declarations in the Dockerfile. But no harm leaving it there.